config/manifests/bases/shipwright-operator.clusterserviceversion.yaml

```diff
@@ -40,7 +40,7 @@ spec:
 
     ```yaml
     ---
-    apiVersion: operator.shipwright.io/v1alpha1
+    apiVersion: operator.shipwright.io/v1beta1
```

Comment:
This should stay as `v1alpha1` - we are not implementing conversion webhooks for the operator API, just the core "build" APIs in the `shipwright.io` group.

Reply:
Agree, This is a mistake